Resources/services/components/applepay_direct.xml

```diff
@@ -16,10 +16,13 @@
 
         <service id="mollie_shopware.components.apple_pay_direct.services.button_builder"
                  class="MollieShopware\Components\ApplePayDirect\Services\ApplePayButtonBuilder" public="true">
+            <argument type="service" id="mollie_shopware.components.account.account" />
             <argument type="service" id="mollie_shopware.config"/>
             <argument type="service" id="config"/>
             <argument type="service" id="mollie_shopware.components.apple_pay_direct.services.payment_method"/>
             <argument type="service" id="mollie_shopware.components.apple_pay_direct.services.display_option"/>
+            <argument type="expression">container.get('Shopware_Components_Modules').Admin()</argument>
```

Comment:
sure that works in showare 5.2.21?

Reply:
I'm not sure to be honest. It's something Shopware uses itself, so I thought it should be okay to use. I tried to pick the less worse option. The Shopware singleton isn't really the best way of doing things either. I'll check what works.

Reply:
could you please try, thank you

Reply:
Well, you were right. Expressions do not work in Shopware 5.2.

I have a little trouble finding the right solution. Using the singleton works. It's also the best supported solution throughout many Shopware 5 version. But than the tests won't work, because the singleton isn't available without booting Shopware. I'm not sure what's the best way to deal with this. I have a solution, but **I would really like to talk it over before it is merged.**